internal/pkg/cli/flag.go

```diff
@@ -364,7 +364,9 @@ Must be of the format '<keyName>:<dataType>'.`
 	storageLSIConfigFlagDescription = `Optional. Attribute to use as an alternate sort key. May be specified up to 5 times.
 Must be of the format '<keyName>:<dataType>'.`
 	storageAuroraServerlessVersionFlagDescription = `Optional. Aurora Serverless version.
-Must be either "v1" or "v2".`
+For "environment" lifecycle, specify "v2". 
+For "workload" lifecycle, specify "v1"/"v2".
```

Comment:
nit: can we replace "/" with the following:
```suggestion
For "workload" lifecycle, specify "v1" or "v2".
```
My small concern is that someone will type `--serverless-version v1/v2`

Reply:
done!! the weird "/" was an attempt to keep the sentence in one line 😂 see if you like the current edition better!